docs/modules/ROOT/pages/explanation/organizations-and-rbac.adoc

```diff
@@ -6,11 +6,17 @@ The organization is also reflected as a `Group` object in the {zone} and is used
 
 == Default RBAC rules
 
-A `RoleBinding` with the name `admin` is created in the `Namespace` which grants the creator the `ClusterRole` called `admin`.
-Another `RoleBinding` with the name `organization-admin` is created in the `Namespace` which also grants the organization `admin` cluster role.
-The user is free to modify or delete either.
+On `Namespace` creation, a number of RBAC rules are created.
+
+* A `RoleBinding` called `admin` which grants the creator of the `Namespace` the `ClusterRole` called `admin`.
```

Comment:
This is AFAIK not done?

Reply:
Not sure about that.

Reply:
This should still be the case. We decided that there should be 2 rolebindings in new namespace: One `admin` for the creator which is generated by the OpenShift project, and an additional one that grants Org admin which is generated by Kyverno

Reply:
It's possible that this is only happening when you create a namespace via OCP mechanisms (web console or `oc new-project`), but not when you create a namespace directly with `kubectl create ns`.

Reply:
Hm, right, we should follow up on that...

Reply:
As discussed with @corvus-ch, the admin RoleBinding is AFAWK only a hack needed when creating the namespace through a project. It's not really needed otherwise, so for now we will just remove the reference here, as it is not guaranteed to exist.

Reply:
We went back and forth in [APUB-84 (internal)](https://ticket.vshn.net/browse/APUB-84) but decided to have separate rolebindings created by the project template and Kyverno in the end.

Reply:
Hm, not creating the `admin` rolebinding feels like an inconsistent user experience. Currently, the `admin` rolebinding is present depending whether `oc new-project` or `kubectl create ns` is invoked. If a user would like to modify or delete the `admin` rolebinding in one namespace, but in other cases (where the ns got created with `kubectl create ns`), the `admin` rolebinding is missing. Users might report a bug on this hidden feature/hack.

There are actual use cases where the `organization-admin` isn't required and actually deleted by the creator. For example, there might be someone "just creating a personal playground namespace to test something" where the organization shouldn't be admin by default. It's easy to overlook the fact that rolebinding `admin` is missing if created by `kubectl` instead of `oc new-project` and then lock themselves out.

My vote would be to create a Kyverno policy that does the same as the project template: Create the `admin` rolebinding like `oc project` would, in case of `kubectl create ns`, but do nothing if it already exists or gets modified.